.gitignore

```diff
@@ -463,3 +463,7 @@ dask-worker-space/
 *.pub
 *.rdp
 *_rsa
+
+# pixi environments
+.pixi
+*.egg-info
```

Comment:
There is already this line in `.gitignore`:

https://github.com/microsoft/LightGBM/blob/ec5492f8f915f9bab99e52417cff6808dbdb8ddd/.gitignore#L299

Is it not working? Why was this `*.egg-info` necessary?